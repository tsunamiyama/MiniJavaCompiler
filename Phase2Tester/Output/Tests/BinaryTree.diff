--- SelfTestCases/BinaryTree.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/BinaryTree.out	2021-02-13 18:30:49.235414500 -0800
@@ -15,12 +15,12 @@
 1
 0
 1
-4
 8
+12
 14
 16
 20
 24
 28
-0
+1
 0
