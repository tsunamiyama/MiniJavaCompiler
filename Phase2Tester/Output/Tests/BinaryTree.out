16
100000000
8
16
4
8
12
14
16
20
24
28
1
1
1
0
1
8
12
14
16
20
24
28
1
0
